CHANGELOG.md

```diff
@@ -48,6 +48,24 @@ _None._
 
 _None._
 
+## 3.3.0
+
+### Breaking Changes
+
+_None._
+
+### New Features
+
+- Make `publish_private_pod` support `--allow-warnings`. [#86]
+
+### Bug Fixes
+
+_None._
+
+### Internal Changes
+
+- Fixed Ruby, RuboCop and RSpec versions on CI
```

Comment:
At first I wondered where that entry came from as I didn't remember seeing a PR for it recently. Then I realized you did this as part of this very PR 😛

```suggestion
- Fixed Ruby, RuboCop and RSpec versions on CI [#88]
```

Reply:
Ah yeah, CI was breaking on this PR and I noticed it was due to using the latest version of RuboCop (potentially a bug, as I've seen it reported for older versions). Updated with this change on 38dc91d68212eb6eeceef7a49ffee06c8191d758.